client/src/components/layout/UserReviewsList.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import UserReviewTile from "./ReviewTile"
+
+const UserReviewsList = ({ reviewsList }) => {
+
```

Comment:
no space padding the first line in a block
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--padded-blocks